extras/noVNC/websockify/websocketproxy.py

```diff
@@ -17,14 +17,14 @@
 try:    from http.server import HTTPServer
 except: from BaseHTTPServer import HTTPServer
 from select import select
-import websocket
+from .websocket import WebSocketRequestHandler, WebSocketServer, ssl
```

Comment:
This does break if you execute this file directly:
```
$ ./extras/noVNC/websockify/websocketproxy.py --help
Traceback (most recent call last):
  File "./extras/noVNC/websockify/websocketproxy.py", line 20, in <module>
    from .websocket import WebSocketRequestHandler, WebSocketServer, ssl
ImportError: attempted relative import with no known parent package
```

However, we don't use that so we don't really care.

I would like to keep the patch small:
```suggestion
from . import websocket
```

Then you can revert the other changes in this file